service/tracing/otel.go

```diff
@@ -4,37 +4,47 @@ import (
 	"context"
 	"log"
 	"os"
+	"strings"
 	"sync"
 
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
 	"go.opentelemetry.io/otel/exporters/stdout/stdouttrace"
+	"go.opentelemetry.io/otel/propagation"
 	"go.opentelemetry.io/otel/sdk/resource"
 	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+	"go.opentelemetry.io/otel/trace/noop"
+	"google.golang.org/grpc/metadata"
 	"gopkg.in/natefinch/lumberjack.v2"
 )
 
-const ServiceName = "github.com/opentdf/platform/service"
+const ServiceName = "opentdf-service"
 
 // Create a thread-safe writer wrapper
 type syncWriter struct {
 	mu     sync.Mutex
 	writer *lumberjack.Logger
 }
 
-type Config struct {
-	Enabled bool   `json:"enabled"`
-	Folder  string `json:"folder"`
-}
-
 func (w *syncWriter) Write(p []byte) (int, error) {
 	w.mu.Lock()
 	defer w.mu.Unlock()
 	return w.writer.Write(p)
 }
 
+type Config struct {
+	Enabled        bool   `json:"enabled"`
+	Folder         string `json:"folder"`
+	ExportToJaeger bool   `yaml:"exportToJaeger"`
+}
+
 func InitTracer(cfg Config) func() {
```

Comment:
Can you update this to take a context (one is present at the call site in start.go, and probably should have a timeout attached in the event this doesn't connect to a tracer backend), and return `(func(), error)`, so we can handle timeouts/other errors from the `.New` calls, instead of calling `log.Fatal`?